TestBenches/VMRouter_test.cpp

```diff
@@ -7,7 +7,7 @@
 #include "FileReadUtility.hh"
 #include "Constants.hh"
 
-const int nevents = 2;  //number of events to run
+const int nevents = 50;  //number of events to run
```

Comment:
Is there any reason not to run over all 100 events in the memory files? Especially for CI, I think it's best to use all the events we have.

Reply:
I agree, except at the moment, ME does not pass for all 100 events. Alexx is working on a fix. But I will update nevents for the other blocks in the meantime.